src/models/request.ts

```diff
@@ -18,3 +21,23 @@ export enum PlateReasonForIssue {
   ORIGINAL = 'Original',
   MANUAL = 'Manual',
 }
+
+export interface Letter {
+  letterType: LetterType;
+  paragraphID: ParagraphID;
+  letterIssuer: string;
+  letterDateRequested: string;
+}
+
+export enum LetterType {
+  TRL_AUTHORISATION = 'trailer authorisation',
```

Comment:
Can this be acceptance please?
```suggestion
  TRL_ACCEPTANCE = 'trailer acceptance',
```